.github/workflows/release.yaml

```
@@ -62,6 +68,10 @@ env:
   # https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage
   git_lfs: false
 
+  # This only helps with consecutive releases to the same tag (and takes up cache storage space).
```

```suggestion
  # Enabling this setting only helps with consecutive releases to the same tag (and takes up cache storage space).
```

Otherwise it's not clear whether the text refers to the `true` or false `variant`
Same for the other instances if this comment